tests/Unit/PointwiseFunctions/AnalyticSolutions/GrMhd/Test_AlfvenWave.cpp

```diff
@@ -87,41 +87,37 @@ void test_variables(const DataType& used_for_size) {
   const double wavenumber = 2.1;
   const double pressure = 1.3;
   const double rest_mass_density = 0.4;
-  const double adiabatic_exponent = 4. / 3.;
+  const double adiabatic_index = 4. / 3.;
```

Comment:
From what I'm aware, "adiabatic index" refers to n and "adiabatic exponent" refers to gamma, in the polytropic relation P = K * rho ^ gamma = K * rho ^ (n+1)/n. In the SmoothFlow and AlfvenWave solutions, it was my intention to use gamma, the adiabatic exponent.

Reply:
I don't think so. The EOS is an ideal fluid not a polytropic fluid and at least in the Rezolla textbook gamma is called the adiabatic index for an ideal fluid.

Reply:
Yes, that was my mistake, I am in agreement with you :)